samples/bluetooth/central_ht/boards/frdm_rw612.conf

```diff
@@ -1 +1,2 @@
 CONFIG_PM=y
+CONFIG_ENTROPY_GENERATOR=y
```

Comment:
we can't be having this is all these overlays for these boards, should just be done by bluetooth subsystem

Reply:
ah just a revert PR, missed that from the link